docs/src/content/docs/guides/split-payments.mdx

```diff
@@ -0,0 +1,331 @@
+
+## Dependencies
+
+- <LinkOut href='https://github.com/interledger/open-payments/tree/main/packages/open-payments'>
+    Open Payments SDK
+  </LinkOut>
+
+<HttpSig />
+
+## Endpoints
+
+- <LinkOut href='https://openpayments.dev/apis/auth-server/operations/post-request/'>
+    Grant Request
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/wallet-address-server/operations/get-wallet-address/'>
+    Get a Wallet address
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-incoming-payment/'>
+    Create an Incoming Payment
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-quote/'>
+    Create a Quote
+  </LinkOut>
+- <LinkOut href='https://openpayments.dev/apis/resource-server/operations/create-outgoing-payment/'>
+    Create an Outgoing Payment
+  </LinkOut>
+
+<Tabs>
+<TabItem label='TypeScript'>
+<Ts />
+
+## Steps
+
+### 1. Import dependencies
+
+Import `createAuthenticatedClient` from the Open Payments SDK package.
+
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/incoming-payment/incoming-payment-create.ts'
+  chunk='1'
+/>
+
+### 2. Create an authenticated Open Payments client
+
+Create an Open Payments-authenticated client by providing the following properties:
+
+- `walletAddressURL` : your Open Payments-enabled wallet address that your client will use to authenticate itself to one or more Authorization Servers (AS).
+- `privateKey` : the EdDSA-Ed25519 key or preferably the absolute or relative file path to the key that is bound to your wallet address. A public key signed with this private
+  key must be made available as a public JWK document at `{walletAddressUrl}/jwks.json` url.
+- `keyId` : the identifier of the private key and the corresponding public key.
+
+<ChunkedSnippet
+  source='https://raw.githubusercontent.com/interledger/open-payments-snippets/main/grant/grant-outgoing-payment.ts'
+  chunk='2'
+/>
+
+### 3. Get WalletAddresses
+
+Fetch wallet addresses of the buyer, merchant and your platform.
+
+```ts
+const merchantWalletAddress = await client.walletAddress.get({
+  url: 'https://ASEabc.com/merchant'
+})
+
+const customerWalletAddress = await client.walletAddress.get({
+  url: 'https://ASExyz.com/alice'
+})
+
+const platformWalletAddress = await client.walletAddress.get({
+  url: 'https://ASEdef.com/platform'
+})
+```
+
+### 4. Request an Incoming Payment grant for the merchant
+
+Request an `incomingPayment` grant from the merchant's wallet's AS.
+
+```ts
+const grant = await client.grant.request(
+  {
+    url: merchantWalletAddress.authServer
+  },
+  {
+    access_token: {
+      access: [
+        {
+          type: 'incoming-payment',
+          actions: ['list', 'read', 'read-all', 'complete', 'create']
+        }
+      ]
+    }
+  }
+)
+```
+
+### 5. Request an Incoming Payment grant for your platform
```

Comment:
Is step 4 required before step 5 or are they interchangeable? I.e., is there a requirement to request the incoming grant for the merchant before requesting it for the platform?